### ISSUE TITLE
First version of menu for the web

### ISSUE THREAD

**Issue by nilsml** [equinor/energyvision#436]
Query menu builder from #435 

We need to solve the problem of fetching data once with statically regenerated pages - and that's a though one. 

- [x] Fetch data
- [x] Create dummy menu component

**Comment by wenche** [comment]
The challenging part is the data fetching part where we want to fetch data only once. While Next.js probably will solve this in a future update, for now there are some tradeoffs. The current status is described in [this issue.](https://github.com/vercel/next.js/discussions/10949#discussioncomment-958694)

[This blogpost](https://flaviocopes.com/nextjs-cache-data-globally/) describes the recommended solution as of today, where you'll fetch the data once and store them in a cache. However, we think that this deserves a separate issue as it touches upon some other issues related to production environment, plan B ++. As a start, we'll just query for the menu data in the pages using a shared function.

Pros:
- Easier to start with

Cons:
- Will query once for every page
- When the menu is changed, we will have stale data before it's regenerated and this might be a problem for less visited pages
